third_party/BUILD

```diff
@@ -469,6 +469,7 @@ UNNECESSARY_DYNAMIC_LIBRARIES = select({
     "//src/conditions:arm": "*.so *.jnilib *.dll",
     "//src/conditions:linux_aarch64": "*.so *.jnilib *.dll",
     "//src/conditions:linux_ppc": "*.so *.jnilib *.dll",
+    "//src/conditions:openbsd": "*.so *.jnilib *.dll",
```

Comment:
Any idea why FreeBSD doesn't show up here? It's surprising.

Reply:
FreeBSD's Bazel port does [patches `third_party/BUILD`](https://github.com/freebsd/freebsd-ports/blob/f5c3ba467a9cc5e834c157b0364d327425981c24/devel/bazel/files/patch-third_party__BUILD) instead.

I considered adding the following line also:
```
    "//src/conditions:freebsd": "*.so *.jnilib *.dll",
```
.. but haven't so far, to avoid mixing up unrelated changes in a single commit and because @aehlig said nothing about this issue earlier. I can certainly add this tweak if you folks like, though.

Reply:
In the absence of anyone expressing an opinion about this, I'll go ahead and close this thread. Feel free to reopen it if you'd like this to change in this PR.